src/component/actions/SendMsg/SendMsgForm.tsx

```diff
@@ -170,18 +167,16 @@ export class SendMsgForm extends React.Component<SendMsgFormProps> {
             <div>
                 <p>Quick Replies are made into buttons for supported channels</p>
                 <TaggingElement
-                    ref={this.props.onBindWidget}
                     name="Replies"
                     placeholder="Quick Replies"
                     prompt="Enter a Quick Reply"
                     onChange={this.handleUpdateQuickReplies}
                     onCheckValid={this.handleCheckValidReply}
                     onValidPrompt={this.handleValidReplyPrompt}
-                    tags={this.props.form.quickReplies || []}
+                    entry={this.props.form.quickReplies}
```

Comment:
all elements will get a FormEntry object which encapsulates the actual value along with it's validity status. The elements themselves can then be pretty dumb with respect to validation.